Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -15,6 +15,14 @@
 f = open(os.path.join(vcs.prefix,"share","vcs","wmo_symbols.json"))
 wmo = json.load(f)
 
+def isTrue(x):
+    """Test if an unsanitized user input is 'true'."""
+    # I'm just encapsulating existing logic here, but this seems to miss some
+    # other cases users may enter. What about:
+    #    - Non-zero integers other than 1?
+    #    - "yes", "Yes", "Y"?
+    return x in ("y", 1, True)
```

Comment:
This spot too:

https://github.com/UV-CDAT/uvcdat/blob/master/Packages/vcs/Lib/VTKPlots.py#L834

If inputs are sanitized, we shouldn't need special cases for backwards compatibility, right?